Mk 6885: Final hydra documentation clean up and review (#14)
oduameh/hydra: docs/docs/faqs.md

<details>
<summary>What is Hydra?</summary>

> ​​Hydra is a family of layer 2 protocols designed to address network scalability capabilities. Hydra head is the first in this protocol suite, providing the foundation on which to build out further scalability.

Reviewer: ```suggestion
​​Hydra is a family of layer 2 protocols designed to address network scalability capabilities. Hydra Head is the first in this protocol suite, providing the foundation on which to build out further scalability.
```

</details>

<details>

> <summary>Is Hydra head secure?</summary>

Reviewer: ```suggestion
<summary>Is the Hydra Head protocol secure?</summary>
```

> Absolutely. Hydra protocols were born out of IOG research, got peer-reviewed, and are implemented using test-driven development. The Hydra head protocol is a true layer 2 and can fall back directly onto the Cardano layer 1 – hence inheriting the security model of the Cardano blockchain.

Reviewer: ```suggestion
Absolutely. Hydra protocols were born out of IOG research, got peer-reviewed, and are implemented using test-driven development. The Hydra Head protocol is a true layer 2 and can fall back directly onto the Cardano layer 1 – hence inheriting the security model of the Cardano blockchain.
```

</details>

<details>
<summary>So what’s this I have heard about ‘1M TPS’?</summary>

> This has been previously referenced as a theoretical maximum, but the reality is more nuanced. For a start, with Cardano’s ‘transactions within transactions’ EUTXO model, TPS itself isn’t a useful metric. A Hydra head is like a small community within a larger group. Initially, these communities operate independently. So, adding up their metrics to get a total picture isn't accurate. Since Hydra head uses the EUTXO model, they can process transactions simultaneously without conflicts, especially with good networking, which optimizes resource usage. As the project progresses, we're constantly evaluating its real-world performance in terms of throughput and finality. For more details, read <a href="https://example.com/more-info">this</a> blog post and see the latest benchmarking data <a href="https://example.com/latest-data">here</a>.

Reviewer: ```suggestion
This has been previously referenced as a theoretical maximum, but the reality is more nuanced. For a start, with Cardano’s ‘transactions within transactions’ EUTXO model, TPS itself isn’t a useful metric. A Hydra head is like a small community within a larger group. Initially, these communities operate independently. So, adding up their metrics to get a total picture isn't accurate. Since Hydra heads use the EUTXO model, they can process transactions simultaneously without conflicts, especially with good networking, which optimizes resource usage. As the project progresses, we're constantly evaluating its real-world performance in terms of throughput and finality. For more details, read <a href="https://example.com/more-info">this</a> blog post and see the latest benchmarking data <a href="https://example.com/latest-data">here</a>.
```

</details>

<details>

> <summary>Can anyone use the Hydra head protocol?</summary>

Reviewer: ```suggestion
<summary>Can anyone use the Hydra Head protocol?</summary>
```

> Yes, it's designed to be accessible for developers and end users alike, requiring minimal changes to existing applications to integrate with Hydra head. However, it is important to note that Hydra is not a network upgrade and it's not like flipping a switch on Cardano to make it fast - instead, applications need to adopt and build on Hydra heads to benefit from it.

Reviewer: ```suggestion
Yes, it's designed to be accessible for developers and end users alike, requiring minimal changes to existing applications to integrate with the protocol. However, it is important to note that Hydra is not a network upgrade, and it's not like flipping a switch on Cardano to make it fast - instead, applications need to adopt and build on Hydra heads to benefit from it.
```

</details>

<details>

> <summary>When is the Hydra head protocol a good fit?</summary>

Reviewer: ```suggestion
<summary>When is the Hydra Head protocol a good fit?</summary>
```

> The Hydra head protocol is well-suited for situations where a known set of participants know each other well enough to agree on building a network but don’t trust each other enough to manage funds without securing their assets. This security is backed by the possibility of settling disputes on layer 1.

Reviewer: ```suggestion
The Hydra Head protocol is well-suited for situations where a known set of participants know each other well enough to agree on building a network but don’t trust each other enough to manage funds without securing their assets. This security is backed by the possibility of settling disputes on layer 1.
```

</details>

<details>

> <summary>Is the Hydra head protocol a sidechain?</summary>

Reviewer: ```suggestion
<summary>Is the Hydra Head protocol a sidechain?</summary>
```
